logos/dynamic_micro_finance_institution/dynamic_micro_finance_institution.svg

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="146px" height="97px" viewBox="0 0 146 97" enable-background="new 0 0 146 97" xml:space="preserve">  <image id="image0" width="146" height="97" x="0" y="0"
```

Comment:
@Nigus-Abate We expect the SVG to be the vector format of the entity's logo. Embedding the png/jpeg format inside an SVG tag doesn't satisfy the requirement.